src/models/base.js

```diff
@@ -2,6 +2,9 @@ const request = require('request-promise-native');
 const url = require('url');
 const async = require('async');
 const crypto = require('crypto');
+const throttledQueue = require('throttled-queue');
+
+const throttle = throttledQueue(10, 1000);
```

Comment:
Let's have a way to configure requests per second, as it is done now for parallel. Of course, this means also that:

- [ ]  the declaration of throttle needs to be moved inside the base object.

- [ ] the new throttle parameter needs to be added in the configuration

```suggestion
const throttledQueue = require('throttled-queue');
[...]
let throttle = throttledQueue(this._throttle, 1000);
[...]
```